src/app/Pages/LandingPage.tsx

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
 import styled from 'styled-components';
 import image from '../Image/LandingPageImage.png';
+import { HiHome, HiPencilAlt, HiOutlineBookOpen } from 'react-icons/hi';
 
 export default function LandingPage(): JSX.Element {
   return (
     <Container>
       <Topic>MY TRAVEL DIARY</Topic>
+      <StyledDiv>
+        <HiHome size={30} style={{ fill: 'var(--font-main)' }} />
+        <Text>
+          Welcome to your personal travel diary. Here you have the opportunity
+          to record and immortalize your special travel experiences.
+        </Text>
+      </StyledDiv>
+      <StyledDiv>
+        <HiPencilAlt size={30} style={{ fill: 'var(--font-main)' }} />
+        <Text>
+          Here you can enter your destination, the date of your trip, your
+          beautiful memories and a picture of your choice.
+        </Text>
+      </StyledDiv>
+      <StyledDiv>
+        <HiOutlineBookOpen size={30} style={{ fill: 'var(--font-main)' }} />
```

Comment:
i think its possible to also style these with styled components as well. 
just a suggestion, i guess both works :)